sdks/python/apache_beam/runners/interactive/cache_manager.py

```diff
@@ -190,7 +190,14 @@ def __init__(self, cache_dir=None, cache_format='text'):
 
   def size(self, *labels):
     if self.exists(*labels):
-      return sum(os.path.getsize(path) for path in self._match(*labels))
+      matched_path = self._match(*labels)
+      # if any matched path has a gs:// prefix, it must be cached on GCS
+      if 'gs://' in matched_path[0]:
```

Comment:
can `matched_path` be empty? If so, `if matched_path and 'gs://' in matched_path[0]`.

Reply:
It seems that `matched_path` is guaranteed to be non-empty here because we only follow this execution path if `self.exists(*labels)` evaluates to True. The exists method calls `self._match(*labels)` and returns True if the matches are non-empty as shown here:
https://github.com/apache/beam/blob/master/sdks/python/apache_beam/runners/interactive/cache_manager.py#L196